src/System.Runtime.Extensions/tests/System/Random.cs

```diff
@@ -54,13 +54,35 @@ public static void Seeded()
         }
 
         [Fact]
-        public static void ExpectedValues()
+        public static void Seeded_Span()
```

Comment:
I expect you're going to need to separate this test out into its own file that's only built when targeting netcoreapp.  The test project has netstandard configurations, and the new API won't be available there.  Some examples are here: https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj#L35-L44